src/Umbraco.Web.UI.Client/src/common/directives/components/umbavatar.directive.js

```diff
@@ -70,8 +70,8 @@ Use this directive to render an avatar.
 
             function getNameInitials(name) {
                 if (name) {
-                    const notAllowed = /[\[\]\{\}\*\?\&\$\@\!\(\)\%\#]+/g;
-                    var names = name.replace(notAllowed,'').trim().split(' '),
+                    const regex = /[^\p{L}\p{N} ]+/gu;
+                    var names = name.replace(regex,'').trim().split(' '),
```

Comment:
This looks great, but lets keep the sensible naming and use the verbose Unicode character class names:
```suggestion
                    const notAllowed = /[^\p{Letter}\p{Number} ]+/gu;
                    var names = name.replace(notAllowed, '').trim().split(' '),
```